helm_deploy/hmpps-interventions-service/templates/ingress.yaml

```diff
@@ -7,8 +7,16 @@ metadata:
   labels:
     {{- include "app.labels" . | nindent 4 }}
   annotations:
-    kubernetes.io/ingress.class: "nginx"
     nginx.ingress.kubernetes.io/custom-http-errors: "418"
+    {{- if .Values.env_details.contains_live_data }}
```

Comment:
Perhaps this should be enabled on all environments so you can deal with any potential false positives sooner.

Reply:
👍 Thanks! I think I will explore that next week -- it would be a bit weird to accidentally block development and user research at this time